sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/BoundedReadEvaluatorFactory.java

```diff
@@ -117,12 +117,19 @@
     private final AppliedPTransform<?, PCollection<OutputT>, Bounded<OutputT>> transform;
     private final InProcessEvaluationContext evaluationContext;
     private boolean contentsRemaining;
+    /**
+     * The source being read from by this {@link BoundedReadEvaluator}. This may not be the same
+     * source as returned by {@link Bounded#getSource()} within {@link #transform} due to splitting.
```

Comment:
Maybe shorten:

```
The source being read from by this {@link BoundedReadEvaluator}.
This may differ from the initial source derived from the {@link #tranfsorm} due to splitting.
```

(Documenting the path by which it is derived seems brittle to future changes)